flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/relational/connection/JdbcConnectionPools.java

```diff
@@ -51,6 +52,26 @@ public HikariDataSource getOrCreateConnectionPool(
             if (!pools.containsKey(poolId)) {
                 LOG.info("Create and register connection pool {}", poolId);
                 pools.put(poolId, jdbcConnectionPoolFactory.createPooledDataSource(sourceConfig));
+            } else {
```

Comment:
> When modifying the time zone connection parameters in the database, the last connection information is still used

when you modify the time zone parameters, then restart the job . The pools(just a map in memory, non persistent) will be empty, why this pr can resolve it?

Reply:
在flink的standalone的session模式下启动会缓存连接池信息，下次启动也不会清空，当然，yarn模式下没有这个问题

Reply:
@lizhongze1  , It seem that flink cdc don't clear the connection pool when job is close.  It will be better if we clear the connection pool when job is close in case of memory leak.

Reply:
Can you please let me know where the code for the Flink CDC connector to trigger related operations when the job is closed?